src/Core/Geographic/Tile.js

```diff
@@ -0,0 +1,256 @@
+import * as THREE from 'three';
+import Coordinates from './Coordinates';
+import CRS from './Crs';
+import Extent from './Extent';
+
+const _dim2 = new THREE.Vector2();
+const _countTiles = new THREE.Vector2();
+const _tmsCoord = new THREE.Vector2();
+const _dimensionTile = new THREE.Vector2();
+const r = { row: 0, col: 0, invDiff: 0 };
+
+function _rowColfromParent(/** @type {Tile} */ tile, /** @type {number} */ zoom) {
+    const diffLevel = tile.zoom - zoom;
+    const diff = 2 ** diffLevel;
+    r.invDiff = 1 / diff;
+
+    r.row = (tile.row - (tile.row % diff)) * r.invDiff;
+    r.col = (tile.col - (tile.col % diff)) * r.invDiff;
+    return r;
+}
+
+const _extent = new Extent('EPSG:4326', [0, 0, 0, 0]);
+const _extent2 = new Extent('EPSG:4326', [0, 0, 0, 0]);
+
+const _c = new Coordinates('EPSG:4326', 0, 0);
+
+export const globalExtentTMS = new Map();
+export const schemeTiles = new Map();
```

Comment:
I think we should take these globals out of the Tile file, but this may be out of scope of this PR. WDYT?

Reply:
Yup agree!
I considered them as a kind of proto-`Tilegrid`. x)